cli/flox-rust-sdk/src/providers/catalog.rs

```diff
@@ -43,7 +44,7 @@ pub const FLOX_CATALOG_DUMP_DATA_VAR: &str = "_FLOX_CATALOG_DUMP_RESPONSE_FILE";
 static GENERATED_DATA: Lazy<PathBuf> =
     Lazy::new(|| PathBuf::from(std::env::var("GENERATED_DATA").unwrap()));
 
-const RESPONSE_PAGE_SIZE: NonZeroU32 = unsafe { NonZeroU32::new_unchecked(10) };
+const RESPONSE_PAGE_SIZE: NonZeroU32 = unsafe { NonZeroU32::new_unchecked(100) };
```

Comment:
nonblocking: would be nice to add some explanation of why 1000, maybe say search `th` returns 42k results

Reply:
And also approximate size of 1k results would be helpful